Cartfile

```diff
@@ -1 +1,2 @@
 github "antitypical/Result" ~> 4.0
+github "pranjalsatija/FutureKit" "220e79165a6d55bda03deffdd716c26e61141513"
```

Comment:
We'll want to make sure it's either a BR Fork or the official FutureKit (which may mean we need to have a discussion on Hyperspace's deployment target.

Reply:
BrightFutures seems to support a low enough version of iOS that we can use it without any problems.

Reply:
I _think_ we're ok to push our deployment target in Hyperspace to 9.0 (possibly even 10) in 4.0, so we might be able to avoid crossing this bridge.

Reply:
BrightFutures works without any changes. I'm going to resolve this because a version update would be in a new PR anyway.